src/events/guildMemberAdd.ts

```diff
@@ -21,10 +21,12 @@ export default class GuildMemberAdd extends Event {
                 if (channel.type !== 'text') return;
 
                 if (settings.logs.join === '--embed') {
+                    const age = Date.now() - member.user.createdTimestamp
                     channel
                         .send(new MessageEmbed()
                             .setColor(0x00ff00)
                             .setAuthor(`${user.tag} (${user.id})`, user.displayAvatarURL())
+                            .setDescription(guild.translate(age < 60000 * 15 ? 'help/logs:NEW_ACCOUNT' : 'help/logs:USER_AGE', { age: guild.client.helpers.convertTime.execute(guild, age) }))
```

Comment:
This is one option, but I'd probably prefer to just create the string first, then pass it into setDescription, cause this shit is hard to read.

```suggestion
                            .setDescription(`${guild.translate('help/logs:USER_AGE', { age: guild.client.helpers.convertTime.execute(guild, age) })} ${age < 60000 * 15 ? guild.translate('help/logs:NEW_ACCOUNT') : '')}`)
```